### ISSUE TITLE
Namespaces in constants not replaced

### ISSUE THREAD

**Issue by danielhuesken** [coenjacobs/mozart#13]
I have found some constant functions in with the Namespaces are not replaced by the new one. Version 0.3.0 is used.

![const_ns](https://user-images.githubusercontent.com/741847/52951326-cb880e80-3381-11e9-8054-0d9a478061c7.png)

**Comment by coenjacobs** [comment]
That's correct. It doesn't replace those (yet). This is already documented in #5, if I recall correctly.

Is this line of code inside the `psr/log` package, or in your own implementation? If it is in your own implementation, you could get around it, by just referring to the constant directly in your code (without the use of the `constant()` function):

```php
PSR\Log\LogLevel::$loggingLevel
```

and Mozart will replace this just fine. If it's in an external package, please provide the (links to) lines of code that it is present in. Thank you!

**Comment by danielhuesken** [comment]
It is in The PayPal SDK https://github.com/paypal/PayPal-PHP-SDK/blob/master/lib/PayPal/Log/PayPalLogger.php#L69

**Comment by coenjacobs** [comment]
The [first beta of the upcoming 1.1.0 release](https://github.com/coenjacobs/mozart/discussions/208) is now available. It would be really helpful if this specific scenario could be tested with this new version, to prove it has been fixed, in a real world scenario. 🙌
